src/components/dataDisplays/cellRenderers/index.js

```diff
@@ -0,0 +1,18 @@
+import DefaultRenderer from './DefaultRenderer';
```

Comment:
I don't understand why this file needs to be an index.js. Is that important?

Reply:
Maybe you _can't_ import multiple files as I stated above. It's just that importing a directory looks for index.js if no file is named explicitly?

Reply:
Exactly!